Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,283 @@
+# Microsoft Mixed Reality Toolkit Release Notes
+
+- [Version 2.0.0](#version-200)
+
+## Version 2.0.0
+
+- [Upgrading projects](#upgrading-projects-to-200)
+- [What's new](#whats-new-in-200)
+- [Known issues](#known-issues-in-200)
+
+This release of the Microsoft Mixed Reality Toolkit 2.0.0 supports the following devices and platforms.
+
+- Microsoft HoloLens 2
+- Microsoft HoloLens (1st gen)
+- Windows Mixed Reality Immersive headsets
+- OpenVR
+
+The following software is required.
+
+- Microsoft Visual Studio (2017 or 2019) Community Edition or higher
+- Windows 10 SDK 18362 or later (installed by the Visual Studio Installer)
+- Unity 2018.4, 2019.1 or 2019.2
+
+### Upgrading projects to 2.0.0
+
+Since the RC2 release, there have been several changes that may impact your application projects,
+including some files moving to new folder locations. For the smoothest upgrade path in your
+projects, please use the following steps.
+
+1. Close Unity
+1. Delete all **MixedRealityToolkit** (you may not have all listed folders)
+    - MixedRealityToolkit
+    - MixedRealityToolkit.Examples
+    - MixedRealityToolkit.Extensions
+    - MixedRealityToolkit.Providers
+    - MixedRealityToolkit.SDK
+    - MixedRealityToolkit.Services
+    - MixedRealityToolkit.Tools
+1. Delete your **Library** folder
+1. Re-open your project in Unity
+1. Import the new unity packages
+    - Foundation - _Import this package first_
+    - (Optional) Tools
+    - (Optional) Extensions,
+    - (Optional) Examples are optional)
```

Comment:
> are optional) [](start = 25, length = 14)

Delete this section (are optional) - it's already specified as optional

Reply:
fixed.